libs/core/src/components/datepicker/datepicker.component.ts

```diff
@@ -103,6 +103,18 @@ class Datepicker extends GdsFormControlElement<Date> {
   @property({ type: Boolean, attribute: 'hide-label' })
   hideLabel = false
 
+  /**
+   * Whether to hide the Clear button under the calendar.
+   */
+  @property({ type: Boolean, attribute: 'hide-clear' })
+  hideClear = false
```

Comment:
@EldRoos I know you're still in draft-mode, but front-running this anyway :)

I think this one should be called `clearable` to align with other components, and thereby also default to `false`. In the future, maybe this should also be moved to a clear-button in the field like in other components as well, but for now, let's align the API-naming at least.